Build Error on Bower - v1.5.0

Hi,

I am having build error with PowerTAC server v1.5.0 on AWS's r3.micro Ubuntu container. The error is with visualizer2, more specifically, it seems to be related to bower. 

My configs:
* 2 vCPU cores
* 1 GB memory
* OS : Ubuntu 18.04
* Bower 1.8.4 

I did the following upon decompression:
```sh
$ cd powertac-server-powertac-server-1.5.0 
$ mvn clean install
```
Then, the following error is returned:
```
[INFO] Running 'bower install --no-color' in /home/ubuntu/powertac/server/powertac-server-powertac-server-1.5.0/visualizer2 
[ERROR] bower bootstrap#3.3.7          EINVRES Request to https://bower.herokuapp.com/packages/bootstrap failed with 502 
[INFO] ------------------------------------------------------------------------ 
[INFO] Reactor Summary: 
[INFO] 
[INFO] Power TAC server modules ........................... SUCCESS [  1.547 s] 
[INFO] Power TAC server interface ......................... SUCCESS [ 11.350 s] 
[INFO] Power TAC accounting service ....................... SUCCESS [  4.270 s] 
[INFO] Power TAC wholesale market ......................... SUCCESS [  2.708 s] 
[INFO] Power TAC intra-hour market-based balancing ........ SUCCESS [  3.053 s] 
[INFO] Power TAC customer models .......................... SUCCESS [  4.229 s] 
[INFO] Power TAC ev customer .............................. SUCCESS [  3.239 s] 
[INFO] Power TAC default broker ........................... SUCCESS [  2.733 s] 
[INFO] Power TAC distribution utility ..................... SUCCESS [  3.029 s] 
[INFO] Power TAC factored customer ........................ SUCCESS [  4.216 s] 
[INFO] Power TAC genco .................................... SUCCESS [  3.789 s] 
[INFO] Power TAC household customer ....................... SUCCESS [ 14.442 s] 
[INFO] Power TAC office complex customer .................. SUCCESS [  8.478 s] 
[INFO] Power TAC logtool core ............................. SUCCESS [  1.460 s] 
[INFO] Power TAC server main .............................. SUCCESS [ 10.071 s] 
[INFO] Power TAC Visualizer v2 ............................ FAILURE [  5.178 s] 
[INFO] ------------------------------------------------------------------------ 
[INFO] BUILD FAILURE 
[INFO] ------------------------------------------------------------------------ 
[INFO] Total time: 01:24 min 
[INFO] Finished at: 2018-10-22T14:57:37Z 
[INFO] Final Memory: 62M/171M 
[INFO] ------------------------------------------------------------------------ 
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.3:bower (bower install) on project visualizer2: Failed to run task: 'bower install --no-color' failed. (error code 1) -> [Help 1] 
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch. 
[ERROR] Re-run Maven using the -X switch to enable full debug logging. 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles: 
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command 
[ERROR]   mvn <goals> -rf :visualizer2 
```
I have also tried `npm install -g bower`, and my bower version is now 1.8.4 (the latest). Then after I re-ran the mvn clean install command, the same error persisted. I even removed the whole directory and rebuilt it, but still got this error.

The link [https://bower.herokuapp.com/packages/bootstrap](https://bower.herokuapp.com/packages/bootstrap) shows this bower version is depreciated, and so did it say so after I update bower. 

```
npm WARN deprecated bower@1.8.4: We don't recommend using Bower for new projects. Please consider Yarn and Webpack or Parcel. You can read how to migrate legacy project here: https://bower.io/blog/2017/how-to-migrate-away-from-bower/
/usr/local/bin/bower -> /usr/local/lib/node_modules/bower/bin/bower 
/usr/local/lib 
└── bower@1.8.4 
```

However, I am able to build and run on my Mac. Would appreciate it if you can help on it. 

Thanks!

--- a/visualizer2/.bowerrc
+++ b/visualizer2/.bowerrc
@@ -1,3 +1,4 @@
 {
-    "directory": "src/main/webapp/bower_components"
+    "directory": "src/main/webapp/bower_components",
+    "registry": "https://registry.bower.io"
 }

